src/server.c

```diff
@@ -1738,7 +1738,7 @@ void beforeSleep(struct aeEventLoop *eventLoop) {
         processed += handleClientsWithPendingWrites();
         processed += freeClientsInAsyncFreeQueue();
         server.events_processed_while_blocked += processed;
-        return;
+        goto lock_pool;
```

Comment:
\i think that `lock_pool` and `el_poll_mutex` should be renamed. at this level (server.c), we don't know if it's `poll` or `select`, so we're just protecting `ae`.

Reply:
changed to `el_mutex` and `lock_el`.